components/homme/test_execs/thetal_kokkos_ut/CMakeLists.txt

```diff
@@ -347,3 +347,4 @@ ELSE()
 ENDIF()
 cxx_unit_test (gllfvremap_ut "${GLLFVREMAP_UT_F90_SRCS}" "${GLLFVREMAP_UT_CXX_SRCS}" "${GLLFVREMAP_UT_INCLUDE_DIRS}" "${CONFIG_DEFINES}" ${NUM_CPUS})
 TARGET_LINK_LIBRARIES(gllfvremap_ut thetal_kokkos_ut_lib)
+cxx_unit_test_add_test(gllfvremap_planar_ut gllfvremap_ut ${NUM_CPUS} "hommexx -planar")
```

Comment:
I thought `cxx_unit_test_add_test' macro is just to go around mpiexec in cxx_unit_test, but here it is called standalone. why?

Reply:
This is to call the same unit test exe twice with different inputs. Before, you could only call it once or else have to create a new exe, meaning another build.